wrangler.toml

```diff
@@ -0,0 +1,23 @@
+# Generated by Wrangler on Mon Aug 05 2024 16:52:18 GMT+0000 (GMT)
+name = "owid"
+compatibility_date = "2024-04-29"
+pages_build_output_dir = "./localBake"
```

Comment:
So, on `owid-cloudflare-prod` this is `~/dist`.
I think we should be fine because we're specifically running `cd dist && npx wrangler pages deploy .` in a deploy, but could you double-check?